src/kohlrahbi/read_functions.py

```diff
@@ -151,3 +152,35 @@ def get_ahb_table(document: Document, pruefi: str) -> Optional[AhbTable]:
     ahb_table.sanitize()
     del seed
     return ahb_table
+
+
+def is_change_history_table(table: Table) -> bool:
+    """
+    Checks if the given table is change history table.
+    """
+    # in the document 'Entscheidungsbaum-DiagrammeundCodelisten-informatorischeLesefassung3.5_99991231_20240401.docx'
+    # I got the error "IndexError: list index out of range", I am not sure which table caused the error
+    try:
+        return table.cell(row_idx=0, col_idx=0).text.strip() == "Änd-ID"
+    except IndexError:
+        return False
+
+
+def get_change_history_table(document: Document) -> Optional[ChangeHistoryTable]:
+    """
+    Reads a docx file and extracts the change history.
+    """
+
+    # Iterate through the whole word document
+    logger.info("🔁 Start iterating through paragraphs and tables")
+    for item in get_all_paragraphs_and_tables(parent=document):
+        style_name = item.style.name  # this is a bit expensive. we should only call it once per item
+        # Check if we reached the end of the current AHB document and stop if it's true.
+        if isinstance(item, Paragraph) and "Änderungshistorie" in item.text and "Heading" in style_name:
+            # checking the style is quite expensive for the CPU because it includes some xpath searches;
+            # we should only check the style if the other (easier/cheap) checks returned True
+            logger.info("🔚 We reached the change history section of the document.")
+
+        if isinstance(item, Table) and is_change_history_table(table=item):
+            change_history_table = ChangeHistoryTable.from_docx_change_history_table(docx_table=item)
+            return change_history_table
```

Comment:
```suggestion
            return change_history_table
        return None
```
explicit ist better than implicit